src/java.base/share/classes/jdk/internal/classfile/impl/StackMapDecoder.java

```diff
@@ -49,29 +51,33 @@
 import jdk.internal.access.SharedSecrets;
 
 import static java.lang.classfile.ClassFile.ACC_STATIC;
+import static java.lang.classfile.ClassFile.ACC_STRICT;
 import static java.lang.classfile.attribute.StackMapFrameInfo.VerificationTypeInfo.*;
 import static java.util.Objects.requireNonNull;
 
 public class StackMapDecoder {
 
     static final int
-                    ASSERT_UNSET_FIELDS = 246,
+                    EARLY_LARVAL = 246,
                     SAME_LOCALS_1_STACK_ITEM_EXTENDED = 247,
                     SAME_EXTENDED = 251;
-    private static final int RESERVED_TAGS_UPPER_LIMIT = ASSERT_UNSET_FIELDS; // not inclusive
+    private static final int BASE_FRAMES_UPPER_LIMIT = SAME_LOCALS_1_STACK_ITEM_EXTENDED; // not inclusive
```

Comment:
shouldn't this be EARLY_LARVAL now?

Reply:
Oh, I moved enclosing frames to another system, so this now intentionally rejects early larval.